tensorflow/core/BUILD

```diff
@@ -274,7 +274,7 @@ cc_library(
         "platform/platform.h",
         "platform/protobuf.h",
         "platform/types.h",
-    ] + glob(tf_additional_proto_hdrs()) + glob(tf_env_time_hdrs()),
+    ] + tf_additional_proto_hdrs() + glob(tf_env_time_hdrs()),
```

Comment:
why remove the glob?

Reply:
`glob` cannot accept list mixed with `select` (according to Bazel error message). This glob does not do anything anywhere since these is no wildcard pattern in `tf_additional_proto_hdrs`. No functionality change for non-Windows.

Reply:
I see. Oh well.